muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/guider/body/application/students/student.tsx

```diff
@@ -6,13 +6,15 @@ import { i18nType } from '~/reducers/base/i18n';
 import { StatusType } from '~/reducers/base/status';
 import { StateType } from '~/reducers';
 import { connect, Dispatch } from 'react-redux';
-
+import moment from "~/lib/moment";
```

Comment:
Could we use newer `import * as moment from 'moment';` which also contains types etc?

Reply:
I  wonder is it makes any difference anymore truly as moment.js is done meaning there wont be any new development happening for it. We will have latest moment.,js (2.29.1) included in the devel when #5861 is reviewed and merged.